hooks/pre-push

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
+
+set -e
```

Comment:
Exit immediately if a pipeline, which may consist of a single simple command, a list, or a compound command returns a non-zero status. [Docs](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin)